pytype/tools/merge_pyi/test_data/imports_multiple.pep484.py

```diff
@@ -1,5 +1,5 @@
 """Test multiple from-imports."""
 from typing import List, Text
 
-def f() -> List[Text]:
+def f() -> List[str]:
```

Comment:
Could you revert this change? This is a test file anyway, and it doesn't really make sense for Text to be imported but not used.